config/brexit_campaign_buckets/cy.yml

```diff
@@ -9,6 +9,9 @@
     Mae'n rhaid i chi hefyd wirio beth sydd angen i chi ei wneud nawr os ydych yn <a href="/prepare-import-to-uk-after-brexit" data-ecommerce-row="true" data-ecommerce-content-id="db1149f5-f60a-4d02-be0c-9c9db2828665" datecoa-ecommerce-path="/prepare-import-to-uk-after-brexit">mewnforio nwyddau o’r UE</a> neu'n <a href="/guidance/providing-services-to-any-country-in-the-eu-iceland-liechtenstein-norway-or-switzerland-after-eu-exit" data-ecommerce-row="true" data-ecommerce-content-id="e9dca8c7-04f5-4065-b2ac-70d140512bfa" data-ecommerce-path="/guidance/providing-services-to-any-country-in-the-eu-iceland-liechtenstein-norway-or-switzerland-after-eu-exit">gwerthu gwasanaethau i’r UE.</a>
```

Comment:
Not really something for this PR, but do you know why we've got these `data-ecommerce` attributes in here? The `datecoa-ecommerce-path` also looks a bit weird/wrong, to me :-|.

Reply:
@benthorner We added e-commerce tracking to the landing page when it was first built - I'm not 100% sure if it's still needed but wary of deleting it at the moment without consulting PAs first.

That is wrong, not sure where it came from... Will amend now.